### PR TITLE
Serveuser1

### PR DESCRIPTION
**Pull request by dhawalbhakta-ugam**

### CODE REVIEW

**Review by vamshikega — COMMENTED** [review]
Have you used system to user to create a user session to ececute your query ?

**Comment by dhawalbhakta-ugam** [comment]
Hi Vamshi ,
Yes I used system to create user session,

File: core/src/main/java/com/ugamdemo/core/models/Impl/SystemuserImpl.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        String listofusers = "";
+        Map<String, String> map = new HashMap<>();
+        map.put("path", "/home/users");
+        map.put("property", "jcr:primaryType");
+        map.put("property.value", "rep:User");
+        map.put("p.hits", "selective");
+        map.put("p.properties", "rep:principalName");
+        map.put("type", "rep:User");
+        try {
+
```
Comment (vamshikega) [inline core/src/main/java/com/ugamdemo/core/models/Impl/SystemuserImpl.java:56]: service user session ?

Comment (dhawalbhakta-ugam) [reply core/src/main/java/com/ugamdemo/core/models/Impl/SystemuserImpl.java:56]: yes, its using service user session.

Comment (vamshikega) [reply core/src/main/java/com/ugamdemo/core/models/Impl/SystemuserImpl.java:56]: you are not using ResolverUtil to get the service user session, check that

**Review by vamshikega — APPROVED** [review]

**Pull request merged**